docs/downloads/index.md

```diff
@@ -64,7 +64,7 @@ Sample Data
 
 Sample data for testing IRIDA can be found at:
 
-<https://irida.corefacility.ca/downloads/data/irida-sample-data.zip>
+<https://sairidapublic.blob.core.windows.net/downloads/data/irida-sample-data.zip>
 
 The sample data package consists of *simulated* Illumina MiSeq read data for three publicly available Listeria Monocytogenes genomes. You can find more information about the simulated data at:
```

Comment:
`README.txt` link below here is still pointing to old url.

Reply:
fixed